.prettierrc.yml

```diff
@@ -0,0 +1,9 @@
+printWidth: 80 # 80
+# tabWidth: 2 # 2
+# useTabs: false # false
+semi: false # true
```

Comment:
same settings as me! haha.
(sorry this isn't an official comment)

Reply:
Pretty sure this is copy pasta'd from other things.